### PR TITLE
Draft of the paper

### PR DESCRIPTION
**Pull request by elswit**
Draft for JOSS submission. Also note the checklist #69 for submission requirements

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    affiliation: "1" # (Multiple affiliations must be quoted)
+  - name: Andrey A. Popov
+    equal-contrib: true # (This is how you can denote equal contributions between multiple authors)
+    orcid: 0000-0002-7726-6224
+    affiliation: 2
+  - name: Arash Sarshar
+    affiliation: 3
+    orcid: 0000-0002-6633-8915
+  - name: Adrian Sandu
+    affiliation: 3
```
Comment (AndreyAPopov) [inline paper/paper.md:27]: Put orc-id

File: paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+affiliations:
+ - name: Lawrence Livermore National Laboratory
+   index: 1
+ - name: Oden Institute for Computational Engineering & Sciences, The University of Texas at Austin
+   index: 2
+ - name: Department of Computer Science and Applications, Virginia Tech
+   index: 3
+
+date: 01 May 2023
```
Comment (AndreyAPopov) [inline paper/paper.md:36]: Is there some way to use the current date?

File: paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+bibliography: paper.bib
+---
+
+# Summary
+
+ODE Test Problems (`OTP`) is an object-oriented OCTAVE/MATLAB package offering a broad range of initial value problems which can be used to test numerical methods such as time integration or data assimilation methods.  It includes problems that are linear and nonlinear, homogeneous and nonhomogeneous, autonomous and nonautonomous, scalar and high-dimensional, stiff and nonstiff, and chaotic and nonchaotic.  Many are real-world problems from fields such as chemistry, astrophysics, meteorology, and electrical engineering.  OTP also supports partitioned ODEs for testing split, multirate, and other multimethods.  Functions for plotting solutions and creating movies are available for all problems, and exact solutions are included when available. OTP is desgined for ease of use---meaning that working with and modifying problems is simple and intuitive.
```
Comment (AndreyAPopov) [inline paper/paper.md:42]: You didn't mention DAE, I think

File: paper/paper.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+A number of existing test problem packages are available in Julia [@rackauckas2017differentialequations] and R with fortran subroutines [@MAZZIA20124119].  While there are some initial value test problems written in Matlab for a variety of scientific applications, they are currently dispersed and not organized into a centralized package with a uniform API.  A well designed open-source collection of test problems for Matlab with Octave compatibility would greatly facilitate numerical method comparison and benchm
+arking across various scientific fields, ultimately leading to the development of more precise and efficient computational methods.
```
Comment (AndreyAPopov) [inline paper/paper.md:52]: why newline?

File: paper/paper.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+# Formulation
+
+All test problems in `OTP` are considered as a first-order  differential-algebraic equation of the form:
+
+$$
+    \mathbf{M}\;Y'(t) = \mathbf{F}(t,Y), \qquad
+    Y(0)  = Y_0,
```
Comment (AndreyAPopov) [inline paper/paper.md:60]: Y(t_0) = Y_0

File: paper/paper.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+$$
+
+where $Y(t)$ is the time-dependent solution to the problem, $F(t,Y)$ is the right-hand-side function representing the time-derivative, and $t$ is the independent variable. $\mathbf{M}$ is the mass-matrix for the differential-algebraic system and when the test problem is an ordinary differential equation, $\mathbf{M}$ is the Identity matrix. The initial condition $Y_0$ specifies the value of $Y$ at the initial time $t = 0$.
```
Comment (AndreyAPopov) [inline paper/paper.md:63]: $t = t_0$

File: paper/problems.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+| lotkavolterra      | A set of differential equations modeling population dynamics in predator-prey systems.             |
+| nbody              | A set of differential equations describing the motion of N interacting point masses.               |
+| oregonator         | The Oregonator model is a set of differential equations used to study oscillatory behavior.         |
+| pendulum           | The simple pendulum equation models the motion of a swinging pendulum.                               |
+| protherorobinson   | The Prothero-Robinson system is used to study the stability of numerical schemes.                   |
+| quadratic          | A simple quadratic model used to study numerical methods for ODEs.                                   |
+| robertson          | The Robertson reaction model is a set of differential equations used to study chemical kinetics.     |
+| sanzserna          | A set of differential equations used to study numerical methods for DAEs.                           |
+| torus              | The torus equation models the motion of a point on a torus.                                           |
```
Comment (AndreyAPopov) [inline paper/problems.md:32]: torus? really?

File: paper/paper.md (hunk 1 above)
Comment (elswit) [reply paper/paper.md:27]: it exists, but  is not populated, what do you think ? 
[0000-0002-5380-0103](https://orcid.org/0000-0002-5380-0103)

Comment (AndreyAPopov) [reply paper/paper.md:27]: Yes, I was informed to use that.

**Pull request merged**